torch/onnx/symbolic_opset10.py

```diff
@@ -333,18 +333,17 @@ def dequantize(g, input):
     return sym_help.dequantize_helper(g, input)[0]
 
 
-# https://github.com/pytorch/pytorch/wiki/PyTorch-ONNX-exporter#quantized-model-export
 class Quantized:
+    """
+    https://github.com/pytorch/pytorch/wiki/PyTorch-ONNX-exporter#quantized-model-export
+    """
     domain = "quantized"
 
     # DequantizeLinear was added in opset version 10.
```

Comment:
Move this comment to the doc string and expand to something like:

`In opset 10 because DequantlizeLinear was added to ONNX in that opset`.